src/components/Profile/SocialNetwork/Network/index.tsx

```diff
@@ -1,23 +1,32 @@
 import React, { useContext, useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
 import { Edit } from '../../styled'
 import { ProfileContext } from '../../Context'
 import { Check, PencilAlt, Times } from '@styled-icons/fa-solid'
-import { Input } from '../../styled'
+import FormText from '../../../Form/components/FormText'
+import { FormItemFullWidth } from '../../../Form/styled'
 
 type Props = {
-	address: string
+	address: string | undefined
 	id: string
 }
 
 export const Network: React.FC<Props> = ({ address, id }) => {
 	const { removeNetwork, editNetwork } = useContext(ProfileContext)
+
 	const [element, setElement] = useState<boolean>(true)
-	const [actualAddress, setActualAddress] = useState<string>('')
+
+	const [actualAddress, setActualAddress] = useState<string | undefined>()
 
 	useEffect(() => {
 		setActualAddress(address)
 	}, [address])
 
+	const { errors, register, handleSubmit } = useForm<FormData>()
+
+	const changeText = (actualAddress: string | undefined) =>
```

Comment:
There's some problem with this. :thinking: 
![ActualAdress](https://user-images.githubusercontent.com/76131194/111833473-82a13c00-88f2-11eb-9bf8-52ce37f91135.png)

Reply:
It doesn't show me any mistakes, so I don't know 🤔

Reply:
That might be a question for @macajind